go.mod

```diff
@@ -39,3 +39,5 @@ replace (
 	k8s.io/metrics => k8s.io/metrics v0.0.0-20190620085625-3b22d835f165
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.0.0-20190620085408-1aef9010884e
 )
+
+go 1.13
```

Comment:
Not sure why `go mod tidy` does this, maybe it should be left out?

Reply:
if go mod tidy wants this... I would leave it.

Reply:
Agreed.